src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -1150,6 +1150,32 @@ public void CZ_ITEM_SELL(ChannelConnection conn, Packet packet)
 		}
 
 		/// <summary>
+		/// Indicates a request from the client to trade with another character.
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="packet"></param>
+		[PacketHandler(Op.CZ_EXCHANGE_REQUEST)]
+		public void CZ_EXCHANGE_REQUEST(ChannelConnection conn, Packet packet)
+		{
+			var targetHandle = packet.GetInt();
+		}
+
+		/// <summary>
+		/// Specifies various commands found in 'customcommand.ies'.
```

Comment:
I'd like it if we can add this server data file in the near future. It is necessary for me to continue the work on quests.

Reply:
Done.